components/Layout.js

```diff
@@ -1,25 +1,27 @@
 import Head from 'next/head'
 import Footer from './Footer'
 import NavBar from './NavBar'
-import styles from '../styles/Home.module.css'
+import styles from '../styles/Home.module.scss'
 
 const Layout = ({ children }) => {
-    return (
-        <>
-            <Head>
-                <title>Course Chart</title>
-                <link rel="icon" href="/favicon.ico" />
-            </Head>
-            
-            <div className={styles.container}>
-                <NavBar />
+  return (
+    <html lang="en">
+      <Head>
+        <title>Course Chart</title>
+        <link rel="icon" href="/favicon.ico" />
```

Comment:
Is this icon our 'CourseChart' icon or the Next.js boilerplate one?

Reply:
It's currently the Next.js boilerplate icon! I'll add the CourseChart icon/.ico to my list though bc that should be easy enough to put together.